.github/workflows/push-to-testnet-draft.yml

```diff
@@ -41,19 +41,19 @@ jobs:
         if: matrix.sbc == 'rockpi' && success()
         with:
           balena_api_token: ${{secrets.BALENA_API_TOKEN}}
-          balena_command: "deploy nebraltd/helium-testnet${{ env.ROCKPI }} --logs --debug --nocache --build --draft"
+          balena_command: "deploy nebraltd/helium-light-testnet${{ env.ROCKPI }} --logs --debug --nocache --build --draft"
```

Comment:
What about consolidating all of these into a single `alena_command: "deploy nebraltd/helium-light-testnet-${{ matrix.sbc }} --logs --debug --nocache --build --draft"`?

Reply:
Why do we need more than one testnet for light? Probably overkill just for testing

Reply:
Also it should be helium-light-test fleet that already exists

Take a look at the light-hotspot-software repo

Reply:
> What about consolidating all of these into a single alena_command: "deploy nebraltd/helium-light-testnet-${{ matrix.sbc }} --logs --debug --nocache --build --draft"?

That could be possible. Let me think about our reasons to keep it separate.

Reply:
It was my idea. We'll need to have separate fleets for every device type. I thought it has to mimic the same environment for testing. Also this is a one time setup. Otherwise, we'll be chasing different things for each device on testnet. That could need more effort than this one. At least this was my humble opinion.

Reply:
> Also it should be helium-light-test fleet that already exists
>
> Take a look at the light-hotspot-software repo

OK. I can use helium-light-test. But AFAIK we had abondoned light-sotspot-software and using light-hotspot branch of helium-miner-software?

Reply:
OK.